docs/development/new-packages.md

```diff
@@ -17,6 +17,18 @@ Most pure Python packages can be installed directly from PyPI with
 {func}`micropip.install` if they have a pure Python wheel. Check if this is the
 case by trying `micropip.install("package-name")`.
 
+Due to [current limitations of webassemly](https://pyodide.org/en/stable/usage/wasm-constraints.html),
```

Comment:
Could you separate the documentation update into a different PR?

Reply:
Sure!

Reply:
I reverted the last commit and added in https://github.com/pyodide/pyodide/pull/5287